azext_iot/central/params.py

```diff
@@ -54,3 +54,9 @@ def load_central_arguments(self, _):
             "MUST INCLUDE type (e.g. 'SharedAccessToken ...', 'Bearer ...'). "
             "Example: 'Bearer someBearerTokenHere'",
         )
+        context.argument(
```

Comment:
I recall another argument `--central-api-uri` that had a similar description. Is this unique from that?